app/models/draw.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+#
+# Model to represent Housing Draws.
+#
+# @attr [String] name The name of the housing draw -- e.g. "Junior Draw".
+# @attr [Integer] year The year of the housing draw -- e.g. 2016.
```

Comment:
Lol I mean to add it in the name within the comment, not to add a separate attribute. Sorry for the confusion, let's revert to avoid unnecessary complexity.